vdoctl/cmd/compat.go

```diff
@@ -27,7 +44,38 @@ var compatCmd = &cobra.Command{
 	Short: "Compatibility matrix of VDO",
 	Long:  `This command helps to configure compatiblity matrix for VDO`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("compat called")
+		ctx := context.Background()
+
+		config, err := buildConfig()
+		if err != nil {
+			panic(err)
+		}
+
+		client, err := runtimeclient.New(config, client.Options{
+			Scheme: scheme.Scheme,
+		})
+		if err != nil {
+			panic(err)
+		}
+
+		err = CreateNamespace(client, ctx)
```

Comment:
Suggestion : First validate the input and then create any resource ( First take the input for matrix, if the matrix is valid then go ahead and create namespace and configmap)